index.bs

```diff
@@ -2634,8 +2637,16 @@ Multiple [=constructor operations=] may appear on a given [=interface=].
 For each [=constructor operation=] on the [=interface=], there will be a way to attempt to
 construct an instance by passing the specified arguments.
 
-The prose definition of a [=constructor operation=] must either initialize the value passed as
-<b>[=this=]</b>, or throw an exception.
+The <dfn>constructor steps</dfn> of a constructor |constructor| should be introduced using text of
+the form “The <code>new |constructor|()</code> constructor steps are:” followed by a list or “The
+<code>new |constructor|()</code> constructor steps are to” followed by an inline description.
```

Comment:
Ah, seems like there's a bit more to do.

```suggestion
The <dfn>constructor steps</dfn> of a [=constructor operation=] that is a member of an interface
named |interface| should be introduced using text of
the form “The <code>new |interface|(|args|)</code> constructor steps are:” followed by a list or “The
<code>new |interface|(|args|)</code> constructor steps are to” followed by an inline description,
where |args| is the list of arguments that the [=constructor operation=] takes.
```

Key changes:

- Link “constructor operation”
- Use the name of the interface
- Add the possibility of having arguments

Reply:
Thanks Timothy! For arguments I followed what we did for method steps. Hopefully that's clear enough.